### ISSUE TITLE
Prepacked installation : err_address_unreachable

### ISSUE THREAD

**Issue by didli** [Jarli01/xenorchestra_installer#96]
Is there some known (network) issues to run xcp-ng as a virtual machine (KVM) ?
Basically, I wanted to test the prepacked installation on a fresh xcp-ng, but while the script installs the xoce VM just fine,
I'm unable to login into it (I can't even ping it) and the server ip reports  : err_address_unreachable.
I've tried both static and DHCP, with no avail.

Note 1 : the official XOA thingy seems to work just fine
Note 2 : anyway, thank you for the prepacked installation update !

**Comment by Jarli01** [comment]
Not that I'm aware of specifically, but your the second person to have made such a ticket since the prepackage was updated. 

I'll take a look in another lab and see if I have any issues.

**Comment by Jarli01** [comment]
I'm downloading the prepackaged now and should have an answer shortly.

**Comment by Jarli01** [comment]
There appears to be an issue with the updated XVA, you can use the older version if you'd like which can be downloaded here. I'll have to upload the source again. 

Old version link: https://nc.nl.tab.digital/s/Ea69m7QoPxgKZsX/download

**Comment by didli** [comment]
Well... it's not working either ^^'. Same result.

**Comment by Jarli01** [comment]
@didli are you downloading the file and attempting to import it, or are you attempting the import process again. If the latter, I've made no changes to the source listed on the main page.

**Comment by didli** [comment]
Here's what I did. I'm probably doing something wrong, but I can't see it :
- download the file directly with wget -O xoce.xva -c https://nc.nl.tab.digital/s/Ea69m7QoPxgKZsX/download
- modify the script xoce-import.sh to handle the downloaded file directly (my connection is ... flimsy, so I can't use curl since it fails quite often)

xcp-ng :
ip : 192.168.122.10
subn : 255.255.255.0
GW : 192.168.122.1
dns : 192.168.122.1 

vm :
ip : 192.168.122.12
subn : 255.255.255.0
GW : 192.168.122.1
dns : 8.8.8.8

Maybe it's a DNS resolving getting wrong. But I've just added the google dns to xcp-ng and restart the VM : without success.
I can ping 8.8.8.8 without issue from xcp-ng but still can't get a ping 192.168.122.12 from the running VM.
I will try again to import the VM with the same DNS that xcp-ng is using by default.
